playbooks/operationalizing/integration.adoc

```diff
@@ -59,6 +52,28 @@ All put together, a command line API request might look something like:
 curl -kI -H "Authentication Bearer: <token>" -H "Content-Type: application/json" -X GET https://master.ose.example.com:8443/oapi/v1/projects
 ----
 
+=== Client Certificates
+
+Like the *Basic API Request* a client certificate request, contains the following components: 
+
+* Identitiy certificate and key: `/etc/origin/master/admin.crt and /etc/origin/master/admin.key` 
+* Content-type Header: `Content-Type: application/json`
+* HTTP Method: `{GET|POST|PUT|PATCH|DELETE}`
+* API URL: `master.ose.example.com:8443`
+* Path ``/<api>/<version>/`
+** The Kubernetes API path is `/api/v1/`
+** The OpenShift API path is `/oapi/v1/`
+
+It should be note, that the *Authentication Header* is missing from the list above and replaced by a "certificate" and "key". 
+
+When building an API call in this form, its beast to use an https://docs.openshift.com/container-platform/latest/admin_guide/allocating_node_resources.html#system-resources-reported-by-node[example] that shows / highlights actual data, like that stats endpoint for a node.
```

Comment:
Minor typo/grammar

When building an API call in this form, it's best to use an

Reply:
Also `s/like that/like the/`.